sdk/keyvault/azure-security-keyvault-keys/src/samples/java/com/azure/security/keyvault/keys/cryptography/CryptographyClientJavaDocCodeSnippets.java

```diff
@@ -186,37 +185,59 @@ public void decrypt() {
         CryptographyClient cryptographyClient = createClient();
-        byte[] signature = new byte[100];
+
         // BEGIN: com.azure.security.keyvault.keys.cryptography.CryptographyClient.sign#SignatureAlgorithm-byte
         byte[] data = new byte[100];
         new Random(0x1234567L).nextBytes(data);
         MessageDigest md = MessageDigest.getInstance("SHA-256");
         md.update(data);
         byte[] digest = md.digest();
+
         SignResult signResult = cryptographyClient.sign(SignatureAlgorithm.ES256, digest);
-        System.out.printf("Received signature of length %d with algorithm %s", signResult.getSignature().length,
-            signResult.getAlgorithm().toString());
+
+        System.out.printf("Received signature of length: %d, with algorithm: %s.\n", signResult.getSignature().length,
+            signResult.getAlgorithm());
         // END: com.azure.security.keyvault.keys.cryptography.CryptographyClient.sign#SignatureAlgorithm-byte
 
         // BEGIN: com.azure.security.keyvault.keys.cryptography.CryptographyClient.sign#SignatureAlgorithm-byte-Context
-        byte[] plaintextData = new byte[100];
-        new Random(0x1234567L).nextBytes(plaintextData);
-        MessageDigest messageDigest = MessageDigest.getInstance("SHA-256");
-        messageDigest.update(data);
-        byte[] digetContent = messageDigest.digest();
-        SignResult signResponse = cryptographyClient.sign(SignatureAlgorithm.ES256, digetContent);
-        System.out.printf("Received signature of length %d with algorithm %s", signResponse.getSignature().length,
-            signResponse.getAlgorithm().toString(), new Context(key1, value1));
+        byte[] dataToVerify = new byte[100];
+        new Random(0x1234567L).nextBytes(dataToVerify);
+        MessageDigest myMessageDigest = MessageDigest.getInstance("SHA-256");
+        myMessageDigest.update(dataToVerify);
+        byte[] digestContent = myMessageDigest.digest();
+
+        SignResult signResponse = cryptographyClient.sign(SignatureAlgorithm.ES256, digestContent,
+            new Context(key1, value1));
+
+        System.out.printf("Received signature of length: %d, with algorithm: %s.\n", signResponse.getSignature().length,
+            signResponse.getAlgorithm());
         // END: com.azure.security.keyvault.keys.cryptography.CryptographyClient.sign#SignatureAlgorithm-byte-Context
 
         // BEGIN: com.azure.security.keyvault.keys.cryptography.CryptographyClient.verify#SignatureAlgorithm-byte-byte
-        VerifyResult verifyResult = cryptographyClient.verify(SignatureAlgorithm.ES256, digest, signature);
-        System.out.printf("Verification status %s", verifyResult.isValid());
+        byte[] myData = new byte[100];
+        new Random(0x1234567L).nextBytes(myData);
+        MessageDigest messageDigest = MessageDigest.getInstance("SHA-256");
+        messageDigest.update(myData);
+        byte[] myDigest = messageDigest.digest();
+        byte[] signature = new byte[100];
```

Comment:
code won't work, either build the signature first or leave it out of the scope with a comment prompting user to sign firat.